securedrop/manage.py

```diff
@@ -18,8 +19,11 @@
 from db import db_session, init_db, Journalist
 from management.run import run
 
+logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s')
+log = logging.getLogger(__name__)
 
-def reset():  # pragma: no cover
+
+def reset(args):  # pragma: no cover
```

Comment:
Because it is going to be used in both [i18n](https://github.com/freedomofpress/securedrop/pull/1983) and a [manage clean_tmp bug fix](https://github.com/freedomofpress/securedrop/pull/1850).